src/config/fonts.ts

```diff
@@ -1,4 +1,5 @@
 export const fontConfig = {
+  'tabler-icons': require('../../assets/tabler-icons/2.47.0/tabler-icons.ttf'),
```

Comment:
can you put this to the end please, as fonts in first case are text fonts, and now icons in addition